Sources/tart/Commands/Get.swift

```diff
@@ -22,10 +23,9 @@ struct Get: AsyncParsableCommand {
   func run() async throws {
     let vmDir = try VMStorageLocal().open(name)
     let vmConfig = try VMConfig(fromURL: vmDir.configURL)
-    let diskSizeInGb = try vmDir.sizeGB()
     let memorySizeInMb = vmConfig.memorySize / 1024 / 1024
 
-    let info = VMInfo(CPU: vmConfig.cpuCount, Memory: memorySizeInMb, Disk: diskSizeInGb,
+    let info = VMInfo(CPU: vmConfig.cpuCount, Memory: memorySizeInMb, Disk: try vmDir.sizeGB(), Size: String(format: "%.3f", Float(try vmDir.allocatedSizeBytes()) / 1000 / 1000 / 1000),
```

Comment:
Should we be have an output consistent with the `tart list`?

```suggestion
    let info = VMInfo(CPU: vmConfig.cpuCount, Memory: memorySizeInMb, Disk: try vmDir.sizeGB(), Size: vmDir.allocatedSizeGB()),
```

Reply:
It's now actually more consistent then before. Before the same value was named `Disk` for `get` but `Size` for `list`.

Regarding `Int` vs `Float` for `list` and `get`. IMO `list` doesn't need that much of precision but `get` can benefit.